src/librustc_target/spec/aarch64_apple_darwin.rs

```diff
@@ -0,0 +1,32 @@
+use crate::spec::{LinkerFlavor, Target, TargetOptions, TargetResult};
+
+pub fn target() -> TargetResult {
+    let mut base = super::apple_base::opts();
+    base.cpu = "apple-a12".to_string();
+    base.max_atomic_width = Some(128);
+    base.eliminate_frame_pointer = false;
+    base.pre_link_args.insert(LinkerFlavor::Gcc, vec!["-arch".to_string(), "arm64".to_string()]);
+
+    base.link_env_remove.extend(super::apple_base::macos_link_env_remove());
+    base.stack_probes = true;
```

Comment:
Out of curiosity does this actually affect anything? (AFAIK stack probes aren't implemented and/or don't work on AArch64)

Reply:
I do not know! This is definitely in the copy-pasta area.

I did see that discussion on [the tier 1 Arm RFC](https://github.com/rust-lang/rfcs/pull/2959#issuecomment-660260930). Is there some way I can check functionality? Should I just set it to false to be conservative?

Reply:
This will set `"probe-stack"="__rust_probestack"` as a function attribute in LLVM IR for all generated functions. I believe that ends up doing nothing in LLVM since nothing on AArch64 recognizes that attribute. In effect I think this is basically a no-opt.

I think you can probably just delete this line since stack probes are disabled by default in target specs (as they only work on x86_64/i686). The AArch64 linux target, for example, doesn't enable this.